tools/src/py-list-deps.py

```diff
@@ -86,12 +86,13 @@ def baz():
         # XXX: This doesn't recognize all the the statement types,
         # just the ones that Emissary currently uses in __init__.py
         # files.
-        if isinstance(node, ast.Import) or  isinstance(node, ast.ImportFrom):
+        if isinstance(node, ast.Import) or isinstance(node, ast.ImportFrom):
```

Comment:
It looks like maybe your IDE did some formatting. @LukeShu - was the black formatter setup to run against tools?

Unless this is block CI from going green we should just drop changes from this file.